react/javascript/demo-external-api/package.json

```
@@ -0,0 +1,99 @@
+{
+  "name": "demo-external-api",
+  "version": "0.0.1",
+  "description": "Extension demoing use of External APIs",
+  "main": "dist/bundle.js",
+  "scripts": {
+    "analyze": "export ANALYZE_MODE=static && yarn build",
+    "build": "export BABEL_ENV=build && webpack --config webpack.prod.js",
+    "clean": "rm -rf dist",
+    "develop": "webpack serve --hot --port 8080 --disable-host-check --config webpack.develop.js",
+    "prebuild": "yarn clean",
+    "lint:es": "eslint 'src/**/*.js{,x}' --cache",
+    "lint:es:fix": "eslint 'src/**/*.js{,x}' --cache --fix",
+    "test": "jest"
+  },
+  "author": "yournamehere",
```

Ha an artifact of create looker extension. maybe a feature to make better...
```suggestion
  "author": "Looker",
```